subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformValuesInjectionIntegrationTest.groovy

```diff
@@ -968,6 +968,11 @@ abstract class MakeGreen implements TransformAction<TransformParameters.None> {
             tasks.create('broken', MyTask)
         """
 
+        executer.expectDocumentedDeprecationWarning("Type 'MyTask': Cannot use @CacheableTransform on type. This annotation can only be used with TransformAction types. " +
```

Comment:
I think it is now time to extract this expectation into a method. I think this will also make it easier to turn the deprecations into errors, since we can then easily assert the failure. Let's do it in a follow up PR.

Reply:
Let's do that when we turn them into errors.